### PR TITLE
samples: zigbee: Fix nRF53 DFU by applying correct sysbuild config

### PR DESCRIPTION
**Pull request by AdamZelikNS**
Update sysbuild configuration to fix Zigbee FOTA (multi-image DFU). Applies to nRF53 platform.

### CODE REVIEW

**Review by milewr — APPROVED** [review]

File: samples/zigbee/light_switch/sysbuild_fota.conf [hunk 1]
```diff
@@ -4,6 +4,5 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-SB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD=y
+SB_CONFIG_PARTITION_MANAGER=y
```
Comment (nordicjm) [inline samples/zigbee/light_switch/sysbuild_fota.conf:7]: I don't get why you would be enabling this when it should be enabled by default

Comment (AdamZelikNS) [reply samples/zigbee/light_switch/sysbuild_fota.conf]: Right, enabling PM was redundant here. I've removed it

**Review by nordicjm — APPROVED** [review]

**Review by edmont — CHANGES_REQUESTED** [review]
Please update changelog before merging.

**Review by edmont — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -602,6 +602,7 @@ Zigbee samples
 * :ref:`zigbee_light_switch_sample` sample:
 
   * Added the option to configure transmission power.
+  * Fixed FOTA operation on nRF5340 by correcting configuration.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:605]:
```suggestion
  * Fixed the FOTA configuration for the nRF5340 DK.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 2 above)
Comment (AdamZelikNS) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: 👍 fixed

**Review by wiba-nordic — APPROVED** [review]

**Pull request merged**